spec/models/member_spec.rb

```diff
@@ -3,34 +3,34 @@
 require 'rails_helper'
 
 RSpec.describe Member, type: :model do
-  before do
-    @member = FactoryBot.create(:member)
-    @blank = 'を入力してください'
-    @already = 'はすでに存在します'
-    @chara_limit = 'は10文字以内で入力してください'
-  end
+  let(:member) { Member.new(params) }
+  let(:params) { {} }
+  let(:blank) { 'を入力してください' }
+  let(:already) { 'はすでに存在します' }
+  let(:chara_limit) { 'は10文字以内で入力してください' }
   example 'first_name,last_name,sex,email,passwordがあれば有効な状態であること' do
-    expect(@member).to be_valid
+    the_member = FactoryBot.create(:member)
+    expect(the_member).to be_valid
   end
   example 'first_nameがなければ無効な状態であること' do
-    member = Member.new(first_name: nil)
+    params.merge!(first_name: nil)
     member.valid?
-    expect(member.errors[:first_name]).to include(@blank)
+    expect(member.errors[:first_name]).to include(blank)
```

Comment:
全体的にvalidation_typeを指定した方がいいと思いますよ。翻訳変わったらテストも修正しなきゃいけないんで。
翻訳の確認もしたいのであればベタが気じゃなくて、ymlファイルから取得してください。

```suggestion
    expect(member.errors.details[:first_name].first[:error]).to eq(:blank)
```